client/components/account-balances/index.tsx

```diff
@@ -1,28 +1,265 @@
+														),
+													},
+											  } )
+											: interpolateComponents( {
+													mixedString: __(
+														'The amount of funds available to be deposited. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
+														'woocommerce-payments'
+													),
+													components: {
+														learnMoreLink: (
+															// eslint-disable-next-line jsx-a11y/anchor-has-content
+															<a
+																rel="external noopener noreferrer"
+																target="_blank"
+																href={
+																	documentationUrls.depositSchedule
+																}
+															/>
+														),
+													},
+											  } )
+									}
+								/>
+							}
+						/>
+						<BalanceBlock
+							id={ `wcpay-account-balances-${ tab.currencyCode }-pending` }
+							title={ fundLabelStrings.pending }
+							amount={ tab.pendingFunds }
+							currencyCode={ tab.currencyCode }
+							tooltip={
+								<BalanceTooltip
+									label={ `${ fundLabelStrings.pending } tooltip` }
+									content={
+										tab.pendingFunds < 0
+											? interpolateComponents( {
+													mixedString: __(
+														'{{learnMoreLink}}Learn more{{/learnMoreLink}} about why your account balance may be negative.',
+														'woocommerce-payments'
+													),
+													components: {
+														learnMoreLink: (
+															// eslint-disable-next-line jsx-a11y/anchor-has-content
+															<a
+																rel="external noopener noreferrer"
+																target="_blank"
+																href={
+																	documentationUrls.negativeBalance
+																}
+															/>
+														),
+													},
+											  } )
+											: interpolateComponents( {
+													mixedString: sprintf(
+														_n(
+															'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
+															'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
+															tab.delayDays,
+															'woocommerce-payments'
+														),
+														tab.delayDays
+													),
+													components: {
+														learnMoreLink: (
+															// eslint-disable-next-line jsx-a11y/anchor-has-content
+															<a
+																rel="external noopener noreferrer"
+																target="_blank"
+																href={
+																	documentationUrls.depositSchedule
+																}
+															/>
+														),
+													},
+											  } )
+									}
+								/>
+							}
+						/>
+					</Flex>
+					{ tab.instantBalance && tab.instantBalance.amount > 0 && (
+						<Flex
+							gap={ 0 }
+							className="wcpay-account-balances__instant-deposit"
+						>
+							<InstantDepositButton
+								instantBalance={ tab.instantBalance }
+							/>
+						</Flex>
+					) }
+				</>
+			) }
+		</TabPanel>
```

Comment:
I skimmed this component – assuming it's relatively unchanged from previous `BalanceTabPanel`.